core/src/main/java/com/linecorp/armeria/common/RequestContext.java

```diff
@@ -484,20 +484,36 @@ default Executor makeContextAware(Executor executor) {
 
     /**
      * Returns an {@link ExecutorService} that will execute callbacks in the given {@code executor}, making
-     * sure to propagate the current {@link RequestContext} into the callback execution.
+     * sure to propagate this {@link RequestContext} into the callback execution.
```

Comment:
Could we also refer to the new method you added and explain the differences here?